package.json

```diff
@@ -13,7 +13,7 @@
   },
   "engines": {
     "node": "^14.18",
-    "npm": "^8.0.0"
+    "npm": "^8.3.0"
```

Comment:
I know there was a reason we stuck with 8.0 but I'm not certain what it was. `npm run build` fails with 8.3.0 _but_ updating the dockerfile with 8.3.0 and running docker build from my linux box succeeds. Will want to discuss this with @nick-funk when he returns